tests/test_plugin_loader.py

```
@@ -4463,5 +4591,67 @@ def test_update_existing_repo_to_branch_or_tag_fetch_failure(
+            try:
+                _exec_plugin_module(
+                    spec=spec,
+                    plugin_module=module,
+                    module_name=mod_name,
+                    plugin_dir=os.path.dirname(__file__),
+                )
+            except Exception:
+                # The loader may raise; that is fine, we test namespace cleanup.
+                # The important assertion is that no RuntimeError from
+                # sys.modules corruption is raised.
+                pass  # nosec B110
```

Ruff (0.15.7)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)